.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Frontend
+
+on:
+  push:
+    branches:
+      - 38-test-sonarqube
```

Comment:
aqui não devia ser na main

Reply:
Good catch. I forgot to switch it back once I was done with tests.
Will fix it.